### ISSUE TITLE
Texture Dumping "ignoreAddress = true" broken in 1.17.X

### ISSUE THREAD

**Issue by chrisfand** [hrydgard/ppsspp#19086]
### Game or games this happens in

Any game

### What area of the game / PPSSPP

Tested on PPSSPP v1.17 and 1.17.1 and devbuild, with Vulkan, DX11, and OpenGL. Using Windows 11.
Last build it works on is 1.16.6.

I let PPSSPP create a textures.ini through the settings menu, and checked to make sure "ignoreAddress = true".

The game still dumps textures without zeroing out the first 8 numbers. Some games I had dumped textures for before (while on Windows 10) that had not been modified for months also dumped textures without zeroing out the numbers, despite the "new" folder containing old zeroed out files.

### What should happen

Texture dumps in the "new" folder should have the first 8 numbers zeroed out when "ignoreAddress = true" is set in textures.ini

### Platform

Windows

### Mobile device model or graphics card (GPU)

Nvidia RTX 3080

### PPSSPP version affected

1.17+

### Last working version

1.16.6

### Graphics backend (3D API)

Vulkan

### Checklist

- [X] Test in the [latest git build](https://www.ppsspp.org/devbuilds) in case it's already fixed.
- [X] [Search for other reports](https://github.com/hrydgard/ppsspp/search?q=my+issue&type=issues) of the same issue.
- [X] Try resetting settings or older versions and include if the issue is related.
- [X] Try without any cheats and without loading any save states.
- [X] Include logs or screenshots of issue.

**Comment by hrydgard** [comment]
thanks, will take a look soon.

**Comment by hrydgard** [comment]
Hm, we have a check that if the hash is set to quick, ignoreAddress is ignored. This is becauase the "quick" hash simply is pretty bad, and xxh is recommended instead. This information is logged to the console but not displayed to the user, I'll make it more obvious.

**Comment by chrisfand** [comment]
My hash is set to xxh64 and addresses are still not being ignored though. Here's my settings on PPSSPP v1.18.1 now.

`[options]
hash = xxh64
ignoreMipmap = true
reduceHash = false
ignoreAddress = true`

**Comment by hrydgard** [comment]
Alright, I'll take a look shortly.

**Comment by chrisfand** [comment]
So I think I found the problem. If the "Replace Textures" option is not enabled in PPSSPP settings while a game is starting, the textures.ini file will not be read at all and therefore none of the options inside of it (including ignoreAddress) are read. This had me confused as when I start creating a texture pack for a new game I only enabled the "save textures" option as I didn't have anything ready to replace yet.

I'd prefer it if the textures.ini was still read when "Replace Textures" is off, just without textures being replaced as I have a hotkey to toggle replacement when testing texture packs.
